auth/familie-ba-sak-auth.yaml

```diff
@@ -0,0 +1,35 @@
+auth:
+  - name: TaskRunner
+    loginEndpointAuth:
+      payloadRaw: "name=TaskRunner&grant_type=client_credentials&code=bar&client_id=bar&client_secret=secret"
+  - name: Veileder
+    loginEndpointAuth:
+      payloadRaw: "name=Veileder&grant_type=client_credentials&code=bar&client_id=bar&client_secret=secret"
+  - name: Saksbehandler
+    loginEndpointAuth:
+      payloadRaw: "name=Saksbehandler&grant_type=client_credentials&code=bar&client_id=bar&client_secret=secret"
+  - name: Beslutter
+    loginEndpointAuth:
+      payloadRaw: "name=Beslutter&grant_type=client_credentials&code=bar&client_id=bar&client_secret=secret"
+  - name: Forvalter
+    loginEndpointAuth:
+      payloadRaw: "name=Forvalter&grant_type=client_credentials&code=bar&client_id=bar&client_secret=secret"
+  - name: Kode6
+    loginEndpointAuth:
+      payloadRaw: "name=Kode6&grant_type=client_credentials&code=bar&client_id=bar&client_secret=secret"
+  - name: Kode7
+    loginEndpointAuth:
+      payloadRaw: "name=Kode7&grant_type=client_credentials&code=bar&client_id=bar&client_secret=secret"
+  - name: System
+    loginEndpointAuth:
+      payloadRaw: "name=System&grant_type=client_credentials&code=bar&client_id=bar&client_secret=secret"
+
+authTemplate:
+    loginEndpointAuth:
+        externalEndpointURL: http://localhost:8083/azuread/token
```

Comment:
ah! forgot about that. it might be the issue u were having. we will have to deal with this somehow, but not in this PR